cfme/tests/control/test_bugs.py

```diff
@@ -26,6 +29,104 @@
 ]
 
 
+def create_policy_profile(request):
+    random_string = fauxfactory.gen_alpha()
+    policy = VMControlPolicy(random_string)
+    policy.create()
```

Comment:
Consider using create_policy here and then introspecting the name. to pass to PolicyProfile